TeapotTests/DeliveryQueueTests.swift

```diff
@@ -27,7 +31,7 @@ class DeliveryQueueTests: XCTestCase {
             expectation.fulfill()
         }
 
-        self.waitForExpectations(timeout: 20.0)
+        self.waitForExpectations(timeout: .defaultTimeout)
```

Comment:
why this interval used to be so much longer before ?

Reply:
I‘m not 100% sure anymore. I think it was just some default.